frontend/lib/src/components/core/Block/Block.tsx

```diff
@@ -161,10 +161,24 @@ const BlockNodeRenderer = (props: BlockPropsWithWidth): ReactElement => {
   }
 
   if (node.deltaBlock.column) {
+    let verticalAlignment: "top" | "bottom" | "center" = "top"
+    if (
+      node.deltaBlock.column.verticalAlignment ===
+      BlockProto.Column.VerticalAlignment.CENTER
```

Comment:
nit: Perhaps we can alias for easier reading?

```js
const { column } = node.deltaBlock
const { VerticalAlignment } = BlockProto.Column

if (column.verticalAlignment = VerticalAlignment.CENTER)
...
```

I think it will simplify a little below too.

Reply:
I simplified it in the latest version 👍